wwr/about.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="blah blah bleh">
+    <meta name="author" content="Lucas Caldas da Costa">
+
+    <!-- Heading Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400..900&display=swap" rel="stylesheet">
+
+    <!-- Text Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400..900&family=Fira+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+
+    <!-- Stylesheet -->
+     <link rel="stylesheet" href="styles/rafting.css">
+
+    <title>White Water Rafting</title>
```

Comment:
Please fix the title according to the course audio tool:

![image](https://github.com/user-attachments/assets/2d9773ab-4996-4713-8971-a65fcf99ddce)

You shouldadd "About Us" along with the name of the rafting company in the title, like this:

```html
<title>About Us | Valfenda Rafting</title>
```